pkg/crd/markers/zz_generated.markerhelp.go

```diff
@@ -198,7 +198,7 @@ func (Minimum) Help() *markers.DefinitionHelp {
 	return &markers.DefinitionHelp{
 		Category: "CRD validation",
 		DetailedHelp: markers.DetailedHelp{
-			Summary: "specifies the minimum numeric value that this field can have.",
+			Summary: "specifies the minimum numeric value that this field can have. Negative integers are supported.",
```

Comment:
Note: I didn't add this... maybe somebody forgot to regenerate something?

Reply:
yeah, someone forgot to regenerate, and the testing missed it somehow.